Pocket/pocket-ios
# feat(cursor): ensuring task restores from cursor

## Summary

Ensures that fetch save operation and fetch archive operations resume from where they left off.

## References 
IN-1269

## Implementation Details
* I also broke Fetching tags off from FetchSaves. In another followup PR I will ensure that Tags are loaded as expected based on https://pocket.slack.com/archives/C01UH57EJKD/p1680536765561499

## Test Steps
* Run app on fresh login. 
* Background app during initial load
* Re open app, loading should pick up where it left off, the best way to watch this is to monitor network requests with something like ProxyMan or use CoreData Lab (app from mac app store) to watch the core data store update the sync cursor on the persistent sync task.

## PR Checklist:

## Screenshots
